demisto_sdk/commands/content_graph/interface/neo4j/neo4j_graph.py

```diff
@@ -495,6 +498,29 @@ def remove_server_items(self) -> None:
         with self.driver.session() as session:
             session.execute_write(remove_server_nodes)
 
+    def _has_infra_graph_been_changed(self) -> bool:
```

Comment:
this method shouldn't be in neo4j_graph.py but in graph.py

Reply:
changed